specifyweb/frontend/js_src/lib/components/Molecules/DateElement.tsx

```diff
@@ -33,9 +34,10 @@ export function DateElement({
         const now = new Date();
         const timeDifference = now.getTime() - dateObject.getTime();
 
-        if (timeDifference < 60000) intervalValue = 5000;
-        else if (timeDifference < 360000) intervalValue = 60000;
-        else intervalValue = 36000000;
+        if (timeDifference < 60 * MILLISECONDS)
+          intervalValue = 5 * MILLISECONDS;
+        else if (timeDifference < 6 * MINUTE) intervalValue = MINUTE;
```

Comment:
did you mean `60 * MINUTE` instead of `6 * MINUTE`?
i.e update every minute, until reaches one hour

Reply:
actually, using `HOUR` instead of `60 * MINUTE` would simplify this

Reply:
but, you still need to add `* MILLISECONDS` in either case